Homework3/2.py

```diff
@@ -0,0 +1,16 @@
+# Напишите программу, которая найдёт произведение пар чисел списка. Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+import random
+import math 
+
+size = int(input("Enter list size: "))
+list = []
+result_list = []
+for i in range(size):
+    list.append(random.randint(1,10))
+
+print(list)
+
+for k in range(int(math.ceil(size/2))):
```

Comment:
Абсолютно верно, но можно и без внешней в библиотек и `math`, например, так:
```suggestion
for k in range((size + 1) // 2):
```